framework/tlvf/AutoGenerated/src/tlvf/ieee_1905_1/tlvWscM1.cpp

```diff
@@ -465,6 +465,13 @@ WSC::sWscAttrOsVersion& tlvWscM1::os_version_attr() {
     return (WSC::sWscAttrOsVersion&)(*m_os_version_attr);
 }
 
+bool tlvWscM1::isPostInitSucceeded() {
+    if( !is_vendor_ext_initialized() ){
```

Comment:
Remove the space before the `!`. Remove the space before the `)`. Add a space before the `{`
So:
```
if (!is_vendor_ext_initialized()) {
```

(obviously should be done in tlvf.py)